### ISSUE TITLE
ForRoot error

### ISSUE THREAD

**Issue by GHakopian** [ItsDizzy/angular2-auth#14]
Hi
In the new Angular CLI the ng serve returns an error on AuthModule.forRoot()

ERROR in Error encountered resolving symbol values statically. Calling function 'AuthModule', function calls are not supported. Consider replacing the function or lambda with a reference to an exported function, resolving symbol AppModule in D:/Git/ClientWeb/src/app/app.module.ts, resolving symbol AppModule in D:/Git/ClientWeb/src/app/app.module.ts

the error will go away when you save once anywhere in the project, but it will stop any publish command for production.

**Comment by paulhobbel** [comment]
Oh that's weird, will look into it

**Comment by hsueing3** [comment]
I'm experiencing the same issue. Could this be caused by the upgrade to Angular4?

**Comment by paulhobbel** [comment]
Yes I believe so, I think I need to use it in a different way now.

**Comment by hsueing3** [comment]
Any ETA on a fix? Would love to continue using this service but will need to switch to another solution pretty soon if there are no updates. :(

**Comment by paulhobbel** [comment]
Writing a fix right now

**Comment by ItsDizzy** [comment]
@hsueing3 could i see your AppModule?

**Comment by paulhobbel** [comment]
Ok this issue should be fixed in 0.1.7. Reopen the issue if the bug still exists.

**Comment by paulhobbel** [comment]
Still seems to happen.

**Comment by marcelo-tavares** [comment]
Is there anything new about this issue?

**Comment by mbehrisch** [comment]
I would need some help with this as well!

**Comment by paulhobbel** [comment]
Ok so this module was originally written for angular2
We are now on angular5 I think, correct me if I am wrong.

Does anyone know how forRoot works in Modules now? I also noticed the ReflectiveInjector is deprecated now... loads of stuff to fix.
